lib/puppet/provider/package/dpkg.rb

```diff
@@ -162,4 +163,12 @@ def unhold
     end
   end
 
+  def package_not_installed?(name)
+    if !name.nil? && !name.empty?
+      output = dpkgquery("-W","-f='${Status} ${Version}\n' #{name}")
+      installed = output.include?('no packages found matching')
```

Comment:
you could just return the evaluation of ` output.include?('no packages found matching')` no need to store in another variable

Reply:
yes